pom.xml

```diff
@@ -48,6 +48,7 @@
     <other.site.name>Stable</other.site.name>
     <autoRelease>false</autoRelease>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <ivy.api>13.2.0-SNAPSHOT</ivy.api>
```

Comment:
how should we maintain this property in the future?
I think we should either leave a note on it in the README.md or implement the ivy-raise script to maintain.